includes/wp-admin/class-settings.php

```diff
@@ -491,6 +518,11 @@ public static function add_screen_option( $screen_settings, $screen ) {
 				<input name="activitypub_show_welcome_tab" type="checkbox" id="activitypub_show_welcome_tab" value="1" ' . \checked( 1, \get_user_meta( \get_current_user_id(), 'activitypub_show_welcome_tab', true ), false ) . ' />
 				' . \__( 'Welcome Page', 'activitypub' ) . '
 			</label>
+			<label for="activitypub_show_advanced_tab">
+				<input name="activitypub_show_advanced_tab" type="hidden" value="0" />
+				<input name="activitypub_show_advanced_tab" type="checkbox" id="activitypub_show_advanced_tab" value="1" ' . \checked( 1, \get_option( 'activitypub_show_advanced_tab' ), false ) . ' />
+				' . \__( 'Advanced Settings', 'activitypub' ) . '
```

Comment:
```suggestion
				' . \esc_html__( 'Advanced Settings', 'activitypub' ) . '
```
This should also be added in line 513 and 519.